games/ninswitch/game_imagerec.py

```diff
@@ -32,11 +34,31 @@
 ]
 
 
+class CaptureScreen(Switch):
```

Comment:
I'm quite sure that you should never open two `AsyncVideoCapture.create("/dev/video21")`s at the same time, and this is why we should try to save frames with only one capture, even when there are two methods to do so.

Can you try once more with `global`? I think something like this should work (after renaming SAVE_FRAMES to SAVE_ALL_FRAMES to better differenciate these frame saving methods):
```python
save_frame = False

class CaptureScreen(Switch):
    async def on(self, seat=0):
        logging.info(f"\t{seat} | Capturing_Frames down")
        global save_frame
        save_frame = True

    async def off(self, seat=0):
        logging.info(f"\t{seat} | Capturing_Frames up")
        global save_frame
        save_frame = False

# later inside image_rec_main:

    if SAVE_ALL_FRAMES or save_frame:
        cv2.imwrite(f"{SAVE_DIR_PATH}/{i}.jpg", frame)
        logging.info(f"SAVED {i}.jpg")
```